CONTRIBUTING.md

```diff
@@ -92,6 +92,15 @@ There is an instance of the system distro running for every user distro running.
 
 Please keep in mind that the system distro is loaded read-only from it's backing VHD. For example, if you need to install tools (say a debugger or an editor) in the system distro, you want to do this in the Dockerfile that builds the system distro so it gets into the private vhd that you are running. You can dynamically install new packages once your have a terminal into the system distro, but any changes you make will be discarded when WSL is restarted.
 
+## Building a debug version
+
+To build a debug version of the system distro, the docker build argument SYSTEMDISTRO_DEBUG_BUILD needs to be set and passed the value of "on". The following command would substitute the docker build command in step 3.2.2 of the "Build Instructions" section.

```

Comment:
Let's switch to using "true" instead of "on" as this is much more intuitive.

Reply:
Good idea, new commit should be updated.